test/parallel/test-fs-readfile-fd-offset.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+
+const filename = path.join(common.tmpDir, 'readfile.txt');
+const dataExpected = new Array(100).join('a');
```

Comment:
Yeah, I'm still not up to speed with ES6. In my defense, I copied from `test-fs-readfilesync-pipe-large.js`.